data/xml/J79.xml

```diff
@@ -5,335 +5,903 @@
+  <title>Natural Language as a Special Case of Programming Languages</title>
+  <author><first>Geoffrey</first><last>Sampson</last></author>
 </paper>
 
 <paper id="1026">
-  <title><i>Junction Grammer as a Base for National Language Processing</i> (Eldon G Lytel; Dennis Packard; Darl Gibb; Alan K Melby; Floyd H Billing Jr)</title>
+  <title><fixed-case>J</fixed-case>unction <fixed-case>G</fixed-case>rammar as a Base for Natural Language Processing</title>
+  <author><first>Eldon G.</first><last>Lytel</last></author>
+  <author><first>Dennis</first><last>Packard</last></author>
+  <author><first>Daryl</first><last>Gibb</last></author>
+  <author><first>Alan K.</first><last>Melby</last></author>
+  <author><first>Floyd H.</first><last>Billings, Jr.</last></author>
 </paper>
 
 <paper id="1027">
-  <title><i>A Formal Psycholinguistics Model Of Sentence Comprehension</i> (Peter Reimold)</title>
+  <title>A Formal Psycholinguistic Model of Sentence Comprehension</title>
+  <author><first>Peter</first><last>Reimold</last></author>
 </paper>
 
 <paper id="1028">
-  <title><i>Summaries Of Contributions To The First National Conference</i> (A Ljudskanov)</title>
+  <title>Summaries of Contributions to the First National Conference on the Application of Mathematical Models and Computers in Linguistics (Part II: Russian Language Contributions)</title>
+  <author><first>A.</first><last>Ljudskanov</last></author>
 </paper>
 
 <paper id="1029">
-  <title><i><fixed-case>ACL</fixed-case> Officers 1976</i> (Stanley R Petrick), <i><fixed-case>ACL</fixed-case> Executive Committee Meeting</i> (Joshi), <i><fixed-case>ACL</fixed-case> Financial Report</i> (A Hood Roberts), <i>Personal Notes</i>, <i><fixed-case>NSF</fixed-case> Reorganization</i> (Dr Edward Creutz), <i><fixed-case>MT</fixed-case> Latsec Shows in <fixed-case>Z</fixed-case>urich</i> (Mr Herbert Bruderer), <i><fixed-case>XII</fixed-case> th International Congress of Linguistics</i> (Wolfgang U Dressler), <i>Acm Conference Employment Register</i> (Orrin E Taulbee), <i>Compcom 76</i> (Edward E <fixed-case>D</fixed-case>avid Jr; J Presper Eckert), <i>Cybernetics And Systems Research</i> (Schottengasse), <i>Literary and Linguistics Computing </i> (Mrs J M Smith), <i><fixed-case>C</fixed-case>harles S Peirce Society</i> (Carolyn Eisele), <i>Computer Applicaton to Learning</i>, <i>Historical Linguistics</i> (William Christie), <i>Short Annoucement and Reminders</i>, <i><fixed-case>ACLS</fixed-case> Soviet Studies East <fixed-case>E</fixed-case>uropean languages</i>, <i>Special Libraries Association Scholarships</i>, <i>Acknowledgement</i> (<fixed-case>A</fixed-case>llen <fixed-case>K</fixed-case>linger), <i>Computer Generation of Sentences By Systemic Grammar</i> (John Self), <i>Interpretation And Intregation of sentences into A C-net</i> (Th. R Hofmann)</title>
+  <title>The FINITE STRING, Volume 12, Number 5</title>
```

Comment:
It seems to me a better idea that each issue of The FINITE STRING receive a new volume number, e.g.,

- J79-1XXX
- J79-2XXX
- J79-3XXX
 
and so on. This more naturally follows the way other volumes are handled (e.g., long and short papers, and the SRW, tutorial abstracts, etc for ACL), and also gives us some margin in case we decide to pull out another article from a specific issue. What do you think?